Libraries/Opc.Ua.Server/NodeManager/SamplingGroupManager.cs

```diff
@@ -225,6 +224,22 @@ public virtual MonitoredItem CreateMonitoredItem(
             return monitoredItem;
         }
 
+        //calculates a revised queue size based on the application confiugration limits
+        private uint CalculateRevisedQueueSize(bool isDurable, uint queueSize)
```

Comment:
Isn't this the same with https://github.com/romanett/UA-.NETStandard/blob/e0a644f85df4da3f4373c326a508e5f068aeca6b/Libraries/Opc.Ua.Server/Diagnostics/CustomNodeManager.cs#L4113 ? Probably move it as utility method ?